app/controllers/api/v1/comments_controller.rb

```diff
@@ -0,0 +1,27 @@
+class Api::V1::CommentsController < ApplicationController
+  load_and_authorize_resource
+
+  def index
+    @comments = Comment.where(post_id: params[:post_id]).order('created_at')
+    render json: { success: true, data: { comments: @comments } }
+  end
+
+  def create
```

Comment:
- [ ] Currently, the endpoint for creating is returning an unprocessable entity error. Probably because the user_id is supposed to be a required field in the creation of the comment. And currently, your endpoints link doesn't include users. Kindly look into this, please.